### DISCUSSION TITLE
Auto Columns not populating in Field Group

### DISCUSSION THREAD

**Discussion by nunick** [Budibase/budibase#4052] [Help] [answered]
Hi all,

New Budibase user here. I've been enjoying the platform so far and hope to create some useful tools for my team with it.

I'm experiencing a issue right now where when I create a Budibase DB that generates screens in the Design section. When updating the Field Groups from the linked table in "new/row" and "/:id" screens, I can't seem to figure out how to have the Auto Columns populate in the group as well. The Auto Columns includes data that I would normally capture and show to my users, so I wouldn't want to create any duplicate columns.

Is there a setting somewhere to select that will let the Auto Columns populate with the rest of the columns that I add?

Edit: I am experiencing this issue on a self hosted version 1.0.26 on Windows 10.

Thank you.

**Comment by mjashanks** [comment, marked as answer]
Hi @nunick  - welcome!

So, the reason that it does not automatically get add to a field group is because they are not user-editable fields.

However, you can easily display the values of these fields to the user. This only makes sense for existing records, as the values are set in the backend, as the user is being created/updated.

To display...

Add a paragraph or heading component, and set the text value to `{{ Repeater.Your Table.Your Auto Column }}`

![image](https://user-images.githubusercontent.com/3524181/149783079-00683b20-a95d-4a54-86f0-1d8090697a33.png)

**Reply by nunick** [reply]
Thank you @mjashanks ! That worked great.

**Comment by joebudi** [comment]
Hi @nunick  . 
I'm the co-founder of Budibase. Would you be interested in jumping on a quick call to discuss how you are using Budibase, and any feedback you may have? This would mean the world to us, and help us build a better product 👍 .
https://calendly.com/budibase/joe
Cheers,
Joe

**Reply by nunick** [reply]
Hi @joebudi, thanks for reaching out! I was able to schedule a call. Looking forward to the chat.
